modules/controller/lib/responders/read.js

```diff
@@ -43,7 +42,7 @@ module.exports = function (err, data, opts) {
     code: 200,
     data: jsonApi(data, {
       one: singleResult,
-      relations: relations
+      relations: data.model.relations
```

Comment:
we only want the relations explicitly requested by the request here, not all possible relations on the model